android/src/main/java/com/robinpowered/Frequency/FrequencyTrackFactory.java

```diff
@@ -0,0 +1,59 @@
+package com.robinpowered.RNFrequency;
+
+import android.media.AudioFormat;
+import android.media.AudioManager;
+import android.media.AudioTrack;
+import android.os.Build;
+
+/**
+ * Created by Ying Hang Eng on 1/24/18.
+ */
+
+class FrequencyTrackFactory {
+    static final int SAMPLE_RATE = 44100;
+
+    public static AudioTrack create(double frequency, double duration) {
```

Comment:
docblock should summarize what this `AudioTrack` does (plays a frequency for a duration)